datasets/ho/make_data.py

```diff
@@ -1,13 +1,16 @@
 from pathlib import Path
 
 import numpy as np
-
+import os
+import sys
 # ========================================= #
 # Set Path here
 # EXAMPLE:
 #   ROOT = '/path/to/HO3D_v2'
 
-ROOT = None
+ROOT = "/mnt/batch/tasks/shared/LS_root/mounts/clusters/wack/code/Users/fho3d"
+sys.stdout.write(f"root is {ROOT}\ncdw is {os.getcwd()}\n")
```

Comment:
Use `print()`.  
Don't use newlines like this just make a new line of code.

```
print(f"root is {ROOT}")
print(f"Current directory is {os.getcwd()}")
```

And this *should* match the code in the other file closer.   It is just extra noise otherwise.   

In better code this would be a common function between the two.